infection.json.dist

```diff
@@ -9,7 +9,8 @@
         "text": "infection.log",
         "badge": {
             "branch": "master"
-        }
+        },
+        "checkstyle": "build/checkstyle.xml"
```

Comment:
Just like it so it could be: 
```json
"github": "/dev/stdout" // or /dev/stderr, whichever works best
```
And the it could output the annotations directly: 
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message

Reply:
If you look in the source of cs2pr, these annotations are pretty damn simple:

https://github.com/staabm/annotate-pull-request-from-checkstyle/blob/79e8cb48fb729d31684d02fe856dbf2282b65d6f/cs2pr#L108-L137

It is not to say we should scrap this new checkstyle report, but it won't exactly work here.

Reply:
I think it can be even better with "github" progress reporter.

Reply:
> I think it can be even better with "github" progress reporter.

what is it?

Reply:
Like ProgressFormatter or DotFormatter. Set with `--formatter` option.